.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: julia
 os:
     - osx
+      osx_image: xcode7
```

Comment:
seriously, coincidence. was browsing on travis to see if re-caching things was going through